README.md

```diff
@@ -4,10 +4,42 @@ This repo contains sample applications written in C# that run on Google Cloud Pl
 
 See [https://cloud.google.com/dotnet/getting-started/hello-world](https://cloud.google.com/dotnet/getting-started/hello-world) for instructions on how to build, run, and deploy the samples.
 
+The applications make use of two client libraries:
+
+* [.NET Cloud Client Library]
+* [Google API Client Library]
+
+## aspnet
+
+The [aspnet] directory contains ASP.NET samples that demonstrate how to access and use Google's APIs from C#. 
+Review this section to learn how use Visual Studio to deploy an ASP.NET sample bookshelf app to the Google Cloud Platform. 
+The sample bookshelf app provides examples of using the following Cloud APIs:
+ 
+* [Cloud Datastore API]
+* [Cloud SQL API]
+* [Cloud Storage API]
+* [Cloud Pub/Sub API]
+
+
+## core
+
+Note: Do Not Build. Do Not Run. This section is currently under development.
+
+The [core] directory contains ASP.NET Core samples that demonstrate how to access and use Google's APIs from C#. 
+
 ## Contributing changes
 
 * See [CONTRIBUTING.md](CONTRIBUTING.md)
 
 ## Licensing
 
 * See [LICENSE](LICENSE)
+
+[.NET Cloud Client Library]: https://github.com/googlecloudplatform/gcloud-dotnet
+[Google API Client Library]: https://github.com/google/google-api-dotnet-client
+[aspnet]: https://github.com/GoogleCloudPlatform/getting-started-dotnet/tree/master/aspnet
+[core]: https://github.com/GoogleCloudPlatform/getting-started-dotnet/tree/master/core
+[Cloud Datastore API]: https://cloud.google.com/datastore/docs/concepts/overview
+[Cloud SQL API]: https://cloud.google.com/sql/docs/admin-api/
+[Cloud Storage API]: https://cloud.google.com/storage/docs/apis
+[Cloud Pub/Sub API]: https://cloud.google.com/pubsub/docs/apis
```

Comment:
For pub/sub and datastore, I like these landing pages better:
https://developers.google.com/api-client-library/dotnet/apis/pubsub/v1
https://developers.google.com/api-client-library/dotnet/apis/datastore/v1beta3
And for Storage:
http://googlecloudplatform.github.io/gcloud-dotnet/index.html

What do you think?